package.json

```diff
@@ -76,21 +81,23 @@
     "eslint-plugin-tsdoc": "^0.2.17",
     "ethers": "^6.7.1",
     "glob": "^10.2.6",
-    "jest": "^29.7.0",
-    "jest-text-transformer": "^1.0.4",
     "nodemon": "^2.0.22",
     "npm-run-all": "^4.1.5",
+    "nyc": "^15.1.0",
     "open": "^8.4.0",
     "prettier": "^3.0.3",
     "rimraf": "^3.0.2",
     "textlint": "^13.3.2",
     "textlint-rule-no-dead-link": "^5.1.2",
     "ts-generator": "^0.1.1",
-    "ts-jest": "^29.1.1",
     "tsup": "^6.7.0",
     "tsx": "^3.12.7",
     "turbo": "^1.8.8",
-    "typescript": "~5.2.2"
+    "typescript": "~5.2.2",
+    "vite": "^4.3.9",
```

Comment:
`vite` is at [v5](https://vitejs.dev/blog/announcing-vite5), so this should be revisited.